examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -80,14 +80,14 @@
   {
```

Comment:
<p>The CI should not redownload the dataset and assume the dataset in the input_data_dir</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>yes, this is how it is implemented in the test, this cell is skipped</p>